pkgs/development/libraries/rocksdb/default.nix

```diff
@@ -28,7 +28,7 @@ stdenv.mkDerivation (finalAttrs: {
     hash = "sha256-bTUzh7ch14TDcm6GkfhA5I/qUVmUm+RE5d2HMZ3zaNc=";
   };
 
-  patches = [ ./fix-findliburing.patch ];
+  patches = lib.optional (lib.versionAtLeast "6.29.3" finalAttrs.version && enableLiburing) ./fix-findliburing.patch;
```

Comment:
I think this should be the other way around:
```suggestion
  patches = lib.optional (lib.versionAtLeast finalAttrs.version "6.29.3" && enableLiburing) ./fix-findliburing.patch;
```

Reply:
Ok, gotcha. Respun.

Reply:
Forgot to push?